### DISCUSSION TITLE
Point Blank doesn't work

### DISCUSSION THREAD

**Discussion by CALR3F** [3kh0/ext-remover#624] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When I try point blank it works for a few seconds but with a lot of lag. After that all tabs reload and the extension comes back.

**Comment by testacc992** [comment]
This wont work because unfortunately this "thing" is glitched. 

![image](https://user-images.githubusercontent.com/94496154/234418513-f914b8cb-fecc-4006-ade9-fd03322b12ea.png)
-Roka

**Reply by CALR3F** [reply]
you mean its patched?

**Comment by BroJac5246** [comment, marked as answer]
What ChromeOS version, blocker extension, and bookmarklet are you using? It could be the result of any of those.

**Reply by CALR3F** [reply]
securly,  version 108,
The bookmarklet is: 
javascript:let shim = false;var ids = prompt("extension ids (comma separated)").split(",");setInterval(()=>{ids.forEach((id)=> opener.chrome.developerPrivate.updateExtensionConfiguration({extensionId: id, fileAccess: shim}));shim = !shim;}, 145);

**Comment by srets** [comment]
im on version 111 and it works for me and my blocker is goguardian
